CSS/divs/chatbot.css

```diff
@@ -1,54 +1,155 @@
-#chatbot-icon {
+@keyframes shake-circle {
+    0% { transform: rotate(0); }
+    25% { transform: rotate(10deg); }
+    50% { transform: rotate(0); }
+    75% { transform: rotate(-10deg); }
+    100% { transform: rotate(0); }
+}
+
+.chatbot-icon {
+    animation: shake-circle 0.5s ease-in-out infinite; /* Apply the circular shake animation */
+}
+
+
+/* Chatbot Icon */
+.chatbot-icon{
+    font-size: 40px;
     position: fixed;
     bottom: 20px;
     right: 20px;
-    background-color: #0078D4;
-    color: white;
+   /* background-color: #66BB6A; /* hot pink */
+    padding: 15px;
     border-radius: 50%;
-    width: 50px;
-    height: 50px;
-    display: flex;
-    align-items: center;
-    justify-content: center;
     cursor: pointer;
-    box-shadow: 0 2px 10px rgba(0, 0, 0, 0.2);
-    z-index: 1000; /* Ensure it appears above other elements */
+    color: white;
+    box-shadow: 0 4px 10px rgba(0, 0, 0, 0.1);
 }
 
-#chat-window {
-    display: none; /* Hidden by default */
+/* Chatbot Container */
+.chatbot-container {
+    width: 300px;  /* Adjusted to fit more content without scrolling */
+    height: 400px;
+    background: url(images/stars.bg.png); /* light pink */
+    border-radius: 15px;
     position: fixed;
-    bottom: 80px; /* Above the icon */
+    bottom: 20px;
     right: 20px;
-    width: 300px;
-    border: 1px solid #ccc;
-    border-radius: 5px;
-    background-color: white;
-    box-shadow: 0 2px 10px rgba(0, 0, 0, 0.2);
-    overflow: hidden;
-    z-index: 1000; /* Ensure it appears above other elements */
+    box-shadow: 0 4px 10px rgba(0, 0, 0, 0.1);
+    display: none;
+    flex-direction: column;
 }
 
-#messages {
```

Comment:
try to make class selector instead of id

Reply:
ohkay noted.